upickle/test/src/upickle/StructTests.scala

```diff
@@ -480,35 +480,35 @@ object StructTests extends TestSuite {
     test("combinations"){
       test("SeqListMapOptionString") - rw[Seq[List[Map[Option[String], String]]]](
         Seq(Nil, List(Map(Some("omg") -> "omg"), Map(Some("lol") -> "lol", None -> "")), List(Map())),
-        """[[],[[[["omg"],"omg"]],[[["lol"],"lol"],[[],""]]],[[]]]""",
+        """[[],[[["omg","omg"]],[["lol","lol"],[null,""]]],[[]]]""",
         upack.Arr(
           upack.Arr(),
           upack.Arr(
-            upack.Obj(upack.Arr(upack.Str("omg")) -> upack.Str("omg")),
+            upack.Obj(upack.Str("omg") -> upack.Str("omg")),
             upack.Obj(
-              upack.Arr(upack.Str("lol")) -> upack.Str("lol"),
-              upack.Arr() -> upack.Str("")
+              upack.Str("lol") -> upack.Str("lol"),
+              upack.Null -> upack.Str("")
             )
           ),
           upack.Arr(upack.Obj())
         )
       )
 
-      test("NullySeqListMapOptionString") - rw[Seq[List[Map[Option[String], String]]]](
-        Seq(Nil, List(Map(Some(null) -> "omg"), Map(Some("lol") -> null, None -> "")), List(null)),
-        """[[],[[[[null],"omg"]],[[["lol"],null],[[],""]]],[null]]""",
-        upack.Arr(
-          upack.Arr(),
-          upack.Arr(
-            upack.Obj(upack.Arr(upack.Null) -> upack.Str("omg")),
-            upack.Obj(
-              upack.Arr(upack.Str("lol")) -> upack.Null,
-              upack.Arr() -> upack.Str("")
-            )
-          ),
-          upack.Arr(upack.Null)
-        )
-      )
+//      test("NullySeqListMapOptionString") - rw[Seq[List[Map[Option[String], String]]]](
```

Comment:
Should we keep this test commented out?

Reply:
This test fails due to the lack of round trippability when mixing Options and Nulls, as described in the PR description. Probably can be kept, but needs to be adjusted to accomodate the new non-round-trip semantics

Reply:
I'd say either adjust it or delete it

Reply:
done